script/Ciel/Day 5/05_01_CIEL05_2_2.txt

```diff
@@ -491,7 +491,7 @@ To a man of the countryside, poverty is a virtue, isn't it?"
 [sha:87133cedcb369ca8adf4f817c4b66149ae53cfba]{
 -- Page 334, Offset 11757.
 -- 「“―――この国に踏み入った理由は、治療と、復讐だ。
-"―――The reasons I set foot in this country are... treatment, and revenge.
+"―――The reasons I set foot in this country are... treatment and revenge.
```

Comment:
pause was intentional. vlov speaks slow af. you could use an ellipses instead i suppose.